junit5/src/main/java/cz/xtf/junit5/annotations/OpenShiftRecorder.java

```diff
@@ -0,0 +1,26 @@
+package cz.xtf.junit5.annotations;
+
+import cz.xtf.junit5.extensions.OpenShiftRecorderHandler;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * {@see OpenShiftRecorderHandler}
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+@ExtendWith(OpenShiftRecorderHandler.class)
+public @interface OpenShiftRecorder {
+
+	/**
+	 * Resource names for filtering out events, pods,...
+	 * Will be turned into regex {@code __value__.*}
+	 *
+	 * {@see OpenShiftRecorderHandler}
+	 */
+	String[] objNames() default "";
```

Comment:
I see two issues with it:
- usability - it would look like `OpenShiftRecorder(objNames=TestClass.APP_NAME + ".")` which does not look very nice
- XTF library works with APP_NAME which is set as name for resources. However there might be different strategies for naming different resources on OpenShift side. For now it just adding some suffix at most (name of pods) but this can change. Hence I would like to keep the control about filtering and regular expressions.

Reply:
Ok, let it be. It can be changed later if not sufficient.